src/ui/components/ObjectView.vue

```diff
@@ -352,6 +352,10 @@ export default {
     show(object, viewKey, immediatelySelect, currentObjectPath) {
       this.updateStyle();
 
+      if (this.domainObject) {
```

Comment:
Instead of this could we add the unsubscribe to `this.triggerStalenessSubscribe`?

Reply:
we can't, we set the domain object a little further down and triggerStalenessSubscribe is called after that, it'd erase the object that was just set.